data/base-shrines.yaml

```diff
@@ -74,7 +74,10 @@ shrines:
     path_out:
     - w
     - out
-  Therenborough:
+  - The altar to Berengaria in the Gash
+    start_room: 2258
+    push: altar
+ Therenborough:
```

Comment:
This indentation looks off to me, is there a space missing?

Reply:
there is indeed a space missing in front of Therenborough: